Contest_API.md

```diff
@@ -900,6 +900,8 @@ JSON elements of problem objects:
 | time\_limit       | number  | no        | no        | Time limit in seconds per test data set (i.e. per single run). Should be an integer multiple of `0.001`.
 | test\_data\_count | integer | yes       | no        | Number of test data sets.
 | max_score         | number  | no        | no        | Maximum expected score, although teams may score higher in some cases. Typically used to indicate scoreboard cell color in scoring contests. Required iff contest:scoreboard_type is `score`.
+| problem_package   | array of FILE  | no | yes    | [Problem package](https://www.kattis.com/problem-package-format/). Expected mime type is application/zip. Only exactly one archive is allowed. Not expected to actually contain href for package during the contest, but used for configuration and archiving.
```

Comment:
I get what you want to say with the sentence, but as a reader I got questions. Like "it's not expected during the contest, but how about before? Or after?" and "When **is** it expected?"

Reply:
For configuration and archiving?

Do you have a suggestion for how to explain it better?

Reply:
Not really. Maybe “for package in live systems” but that also is meh

Reply:
I think what I wrote is better.